joule-labs/joule-extension
# Loop Swap Status

Closes #246

### Loop Swap Status

Added v1/loop/swaps to loop-http lib. Added Swap List in Home Page tab pane with Swap Details.
TODO: Make swap details pretty / code cleanup and review.

### Steps to Test

1. Install LND 0.10.0-beta.rc2
2. Intall LOOP 0.5.1-beta 
3. Set up loop on Loop Out/In settings
4. Initiate Loop / Out In
5. View status on "Swaps" tab
6. Click on Swap Row to see Swap Details

### Screenshots

[Only if applicable]
![image](https://user-images.githubusercontent.com/13033037/79534971-1535bd00-804a-11ea-95fd-f41b38bbb3c3.png)

![image](https://user-images.githubusercontent.com/13033037/79535072-604fd000-804a-11ea-8494-ed8d515fb3ce.png)
